source/agora/flash/Config.d

```diff
@@ -39,7 +40,7 @@ public struct FlashConfig
     public immutable string[] addresses_to_register;
 
     /// Timeout for requests
-    public Duration timeout = 10.seconds;
+    public @fromMsecs Duration timeout = 10.seconds;
```

Comment:
Does the `@fromMsecs` convert the `seconds` to `msecs`?

Reply:
No it converts `msecs` as string from a `config.yaml` to a `Duration`. The default here is specified in `seconds` but could be any valid `Duration`.

Reply:
Understood. Is the `Duration` type specified by calling it (`.seconds`)?

Reply:
@kchulj : A `Duration` is just a `struct` that stores `hnsecs`:
https://github.com/dlang/druntime/blob/32299128cfebc9886703256ccda3654cfa05c76e/src/core/time.d#L1701-L1714

That's all there is. hnsecs stands for hecto-nanoseconds (100 ns). So one millisecond is 10_000 hnsecs, and one second is 10_000_000 hnsecs. So all `seconds` do is return a `Duration` which has the argument times 10M.